### DISCUSSION TITLE
iCal export for all events

### DISCUSSION THREAD

**Discussion by tiefpunkt** [pretix/pretix#2584] [Ideas] [answered]
It would be great to export all events as an iCal feed. I found #389 and #413, but these are for individual events only. I'd like to include all events with our website calendar automatically, and an iCal feed would be the perfect solution for that.

What are your thoughts on that?

Best,
Severin

**Comment by pc-coholic** [comment, marked as answer]
Systemwide iCals probably won't happen unless added by a custom plugin - but perhaps the organizer ical feed is what you're looking for?

Example: https://pretix.eu/demo/events/ical/?locale=de

**Comment by choseh** [comment]
@tiefpunkt feel free to upvote https://github.com/pretix/pretix/discussions/2273 ;)

**Comment by tiefpunkt** [comment]
In this case the organizer ical feed meets my needs, thanks @pc-coholic. Not sure why I didn't see that earlier.

**Comment by mklemme1** [comment]
Hallo

We are just setting up a self-hosted Prefix. 
I did not manage to construct the ICAL URL following the pattern above. 
Can somebody help?
Thanks
